modules/tinymce/CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed extra padding for the context toolbar in the `tinymce-5` skin. #TINY-8980
 - Fixed a regression where pressing Enter caused content outside the selection to be added or deleted unexpectedly. #TINY-9101
 - Fixed a bug where pressing Enter deleted selected "contenteditable="false" pre elements. #TINY-9101
+- The `editor.insertContent()` API did not respect the `no_events` argument. #TINY-9140
```

Comment:
So is the change that we are passing through `no_events` now, or is the change that we are propagating what we are given, and users could pass through custom data if they wanted to ?

Reply:
I guess we aren't testing anything more than `no_events` so maybe that's the documented change, and the other aspect of it is just implementation detail?